### PR TITLE
base16-universal-manager: init at 1.0

### PR DESCRIPTION
**Pull request by jo1gi**
###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [X] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [X] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [X] Ensured that relevant documentation is up to date
- [X] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

**Review by bbjubjub2494 — CHANGES_REQUESTED** [review]
##### Reviewed points

- [x] package path fits guidelines
- [x] package name fits guidelines
- [x] package version fits guidelines
- [x] package build on x86_64-linux
- [x] executables tested on x86_64-linux
- [x] `meta.description` is set and fits guidelines
- [x] `meta.license` fits upstream license
- [x] `meta.platforms` is set
- [x] `meta.maintainers` is set
- [x] build time only dependencies are declared in `nativeBuildInputs`
- [x] source is fetched using the appropriate function
- [x] phases are respected **NA**
- [x] patches that are remotely available are fetched with `fetchpatch` **NA**

##### Possible improvements
- please see review comments
- according to the CI there are tabs in your file.  i suggest running [nixpkgs-fmt](https://github.com/nix-community/nixpkgs-fmt) if you to automate out these concerns.

##### Comments

**Review by bbjubjub2494 — APPROVED** [review]
LGTM

File: pkgs/applications/misc/base16-universal-manager/default.nix [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+  pname = "base16-universal-manager";
+  version = "1.0";
+  src = fetchFromGitHub {
+    owner = "pinpox";
+    repo = "base16-universal-manager";
+    rev = "v${version}";
+    sha256 = "11kal7x0lajzydbc2cvbsix9ympinsiqzfib7dg4b3xprqkyb9zl";
+  };
+  vendorSha256 = "19rba689319w3wf0b10yafydyz01kqg8b051vnijcyjyk0khwvsk";
+  meta = with lib; {
```
Comment (SuperSandro2000) [inline pkgs/applications/misc/base16-universal-manager/default.nix:13]:
```suggestion
buildGoModule rec {
  pname = "base16-universal-manager";
  version = "1.0";

  src = fetchFromGitHub {
    owner = "pinpox";
    repo = "base16-universal-manager";
    rev = "v${version}";
    sha256 = "11kal7x0lajzydbc2cvbsix9ympinsiqzfib7dg4b3xprqkyb9zl";
  };

  vendorSha256 = "19rba689319w3wf0b10yafydyz01kqg8b051vnijcyjyk0khwvsk";

  meta = with lib; {
```

File: pkgs/applications/misc/base16-universal-manager/default.nix [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+    description = "A universal manager to set base16 themes for any supported application";
+    homepage = "https://github.com/pinpox/base16-universal-manager";
+    license = licenses.mit;
+    maintainers = with maintainers; [ jo1gi ];
+    platforms = platforms.unix;
```
Comment (SuperSandro2000) [inline pkgs/applications/misc/base16-universal-manager/default.nix:18]:
```suggestion
```

buildGoModule defaults to the right subset of unix.

File: maintainers/maintainer-list.nix [hunk 3]
```diff
@@ -4735,6 +4735,12 @@
     githubId = 587870;
     name = "Jonathan Mettes";
   };
+  jo1gi = {
```
Comment (SuperSandro2000) [inline maintainers/maintainer-list.nix:4738]: Please add yourself as a maintainer by adding an alphabetically sorted entry to `maintainers/maintainer-list.nix` and commit this change into a separate commit with the message ``maintainers: add jo1gi``.

**Pull request merged**